index.html

```diff
@@ -211,6 +211,44 @@ <h1>About me</h1>
         </div>
       </div>
     </section>
-      
+    <div class="contact-me">
+      <h2 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/meqdprbd" class="contact-form" method="post">
+        <input type="text" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+        <input type="email" class="contact-form-email" placeholder="Email address" required>
+        <textarea name="contact" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
+        <div class="form-button"><input type="submit" value="Get In Touch" class="contact-button"></div>
+      </form>
+      <hr/>
+      <div class="social-icons" id="social-icons">
+        <ul>
+          <li class="socials">
+            <a href="#"><i class="fa fa-github"></i></a>
+          </li>
+          <li class="socials">
+            <a href="#"><i class="fa fa-linkedin"></i></a>
+          </li>
+          <li class="socials">
+              <a href="#"><i class="fa fa-hand-peace-o"></i></a>
+          </li>
+          <li class="socials">
+            <a href="#"><i class="fa fa-twitter"></i></a>
+          </li>
+          <li class="socials">
+            <a href="#"><i class="fa fa-medium"></i></a>
+          </li>
+        </ul>
+      </div>
+
+      <!-- <div class="contact-icon">
+        <img src="./Pictures/SocialMedia2.png" alt="Social media">
+      </div>
+      <div class="shape">
+        <img src="./Pictures/Shape.png" alt="shape">
+      </div> -->

```

Comment:
- [ ] I think it's better not to leave unused commented code on your file. This creates an unprofessional look to your work not suited to show to recruiters and hiring managers.
- [OPTIONAL] Please remove the unused commented code from your CSS file also for the same reason. As not required for the previous reviewer I consider it optional but highly recommended.